rqd/rqd/rqcore.py

```diff
@@ -567,17 +558,16 @@ def __init__(self, optNimbyoff=False):
 
     def start(self):
         """Called by main to start the rqd service"""
-        # If nimby should be on, start it
-        if rqconstants.OVERRIDE_NIMBY:
-            log.warning("Nimby startup has been triggered by OVERRIDE_NIMBY")
-            self.nimbyOn()
-        elif self.machine.isDesktop():
+        if self.machine.isDesktop():
             if self.__optNimbyoff:
                 log.warning("Nimby startup has been disabled via --nimbyoff")
             elif not rqconstants.OVERRIDE_NIMBY:
                 log.warning("Nimby startup has been disabled via OVERRIDE_NIMBY")
```

Comment:
nit: I'm not a huge fan of this comment since it implies that the setting of `OVERRIDE_NIMBY` has disabled nimby, when in actuality the absence is what disabled nimby.

Reply:
I guess it makes sense since it can be set to `False`, but since the default value is `None` it still annoys me.

Reply:
I've added a bit more code here to distinguish between the various modes and log slightly differently in each case. That should hopefully be more helpful.